### PR TITLE
[#48250] implement global index page for boards

### PR DESCRIPTION
**Pull request by klaustopher**
Implements the global index page for boards.

See https://community.openproject.org/projects/openproject/work_packages/48250

### CODE REVIEW

File: modules/boards/app/views/boards/boards/overview.html.erb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+         id="add-board-button"
+         title="<%= I18n.t(:label_meeting_new) %>"
+         arial-label="<%= I18n.t(:label_meeting_new) %>"
+         class="button -alt-highlight">
+        <%= op_icon('button--icon icon-add') %>
+        <span class="button--text"><%= t(:label_meeting) %></span>
+      </a>
+    </li>
+  <% end %>
+<% end %>
```
Comment (klaustopher) [inline modules/boards/app/views/boards/boards/overview.html.erb]: The modal for creating a new board currently does not include any selection of the project. Should we add a WP (like we have for global meetings create) to implement a new modal view that will include a product selection?

//cc @oliverguenther

Comment (klaustopher) [reply modules/boards/app/views/boards/boards/overview.html.erb]: Never mind, I saw that https://community.openproject.org/projects/openproject/work_packages/47989/activity exists

**Review by oliverguenther — CHANGES_REQUESTED** [review]
Only a small thing I noticed: The global page is missing a heading. Apart from that, LGTM :+1:

**Comment by klaustopher** [comment]
Nice catch. Title added

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**